Cargo.toml

```diff
@@ -141,5 +141,16 @@ pallet-hotfix-sufficients = { version = "1.0.0", path = "frame/hotfix-sufficient
 # Frontier Template
 frontier-template-runtime = { path = "template/runtime", default-features = false }
 #GGX
-account-filter = { package = "substrate-account-filter", git = "https://github.com/akorchyn/substrate-account-filter.git", default-features = false, branch = "polkadot-v0.9.35" }
-# pallet-collective = { version = "4.0.0-dev", git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.35", default-features = false }
\ No newline at end of file
+account-filter = { package = "substrate-account-filter", git = "https://github.com/GoldenGateGGX/substrate-account-filter", branch="polkadot-v0.9.35", default-features = false }
```

Comment:
We should consider including this in the `frame/` directory so we don't have to publish it...

Reply:
Agree. But probably, it's better to do in separate PR.

Reply:
As discussed, seems like we'll do this in this PR.